packages/docs/docs/components/toast.mdx

```diff
@@ -45,7 +45,35 @@ Using `toast` hook you can trigger toast from anywhere in your application.
 </Button>
 ```
 
+### Toast With Promise
+Using `toast.promise` you can show toast with promise.
 
+```jsx
+<Button
+variant="outline"
+onClick={() => {
+  //highlight-start
```

Comment:
Is this comment required? Is code highlighting not working without this? Also can you prettify this block?

Reply:
Yes, we need the comment to highlight the code @akzhysparrow